_posts/hannahlwang/2016-01-14-hannahlwang-first-post.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: post
 author: hannahlwang
+title: Hannah's First Post!
```

Comment:
Nice post!  But I believe this `'` needs to be enclosed in `"` so it won't confuse Jekyll.